### PR TITLE
Rewrite qcms fuzz target for the change to Rust

### PR DESCRIPTION
**Pull request by jrmuizel**

### CODE REVIEW

**Comment by alex** [comment]
No substantive review here, just wanted to pop in and say this is super excited that qcms was migrated! It'd be wonderful if there was a good write up of the c2rust and then remove-unsafe process!

**Comment by jrmuizel** [comment]
Yeah, the remove-unsafe work is still happening, but I plan to write something up when it's done.

File: projects/qcms/Dockerfile [hunk 1]
```diff
@@ -15,7 +15,7 @@
 ################################################################################
 
 FROM gcr.io/oss-fuzz-base/base-builder
-RUN apt-get update && apt-get install -y mercurial
+RUN apt-get update && apt-get install -y mercurial make autoconf automake libtool curl cmake python llvm-dev libclang-dev clang
```
Comment (inferno-chromium) [inline projects/qcms/Dockerfile:18]: Do you need 
"make curl cmake python llvm-dev libclang-dev clang"
All of these should be on the image and we should be using the latest clang already provided on image.

**Review by inferno-chromium — COMMENTED** [review]
This is exciting, have a question on DockerFile.

File: projects/qcms/Dockerfile (hunk 1 above)
Comment (jrmuizel) [reply projects/qcms/Dockerfile:18]: I just copied the stuff from `serde_json/Dockerfile` as an example. It's probably not needed. I'll take it out.

File: projects/qcms/build.sh [hunk 2]
```diff
@@ -15,24 +15,9 @@
...
-  -DBUILD_FOR_OSSFUZZ \
-  $LIB_FUZZING_ENGINE \
-  fuzztest/qcms_fuzzer.cpp -o $OUT/fuzz
-
-find fuzztest -iname "*.icc" \
+find fuzz -iname "*.icc" \
   -type f -exec zip -qju $OUT/fuzz_seed_corpus.zip "{}" \;
-cp fuzztest/qcms_fuzzer.dict $OUT/fuzz.dict
+cp fuzz/qcms_fuzzer.dict $OUT/fuzz.dict
+cp fuzz/target/x86_64-unknown-linux-gnu/release/fuzz_target_qcms $OUT/
```
Comment (inferno-chromium) [inline projects/qcms/build.sh:23]: Does this need to be $OUT/fuzz

Comment (jrmuizel) [reply projects/qcms/build.sh:23]: https://google.github.io/oss-fuzz/getting-started/new-project-guide/rust-lang/ suggests that '$OUT/' is sufficient.

Comment (jrmuizel) [reply projects/qcms/build.sh]: Ah. That makes sense.

Comment (jrmuizel) [reply projects/qcms/build.sh]: And I pushed a changed version before seeing your commit

**Pull request merged**